bin/mozinfo

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
JSON doesn't support comments unfortunately.
this mozinfo file is used by both xpcshell and mochitest.
So I let things as-is here.